tools/deviceGen/input/device_features.json

```diff
@@ -0,0 +1,2666 @@
+            ]
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseBinding",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidencyBuffer",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidencyImage2D",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidencyImage3D",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidency2Samples",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidency4Samples",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidency8Samples",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidency16Samples",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "sparseResidencyAliased",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "variableMultisampleRate",
+            "expose": false,
+            "comment": "[EXPOSE AS LIMIT] poor support on Apple GPUs"
+        },
+        {
+            "type": "bool",
+            "name": "inheritedQueries",
+            "value": true,
+            "expose": false,
+            "comment": "[REQUIRE] Always enabled, good device support."
+        }
+    ],
+    "vulkan11core": [
+        {
+            "groupComment": [
+                "Vulkan 1.1 Core"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "storageBuffer16BitAccess",
+            "value": true,
+            "expose": false,
+            "comment": "[REQUIRED] ROADMAP 2024"
+        },
+        {
+            "type": "bool",
+            "name": "uniformAndStorageBuffer16BitAccess",
+            "value": true,
+            "expose": false,
+            "comment": "[REQUIRED] Force Enabled : ALIAS VK_KHR_16bit_storage"
+        },
+        {
+            "groupComment": [
+                "[EXPOSE AS A LIMIT] Enabled by Default, Moved to Limits : ALIAS VK_KHR_16bit_storage"
+            ]
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "storagePushConstant16",
+            "expose": false
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "storageInputOutput16",
+            "expose": false
+        },
```

Comment:
should be in the group AND https://github.com/Devsh-Graphics-Programming/Nabla/pull/643/files#r1641940730